_ext/jboss_developer.rb

```diff
@@ -117,17 +117,22 @@ def transform site, page, content
                     else
                       if (page.metadata[:browse].include?('blob') || page.metadata[:browse].include?('tree'))
                         if (page.metadata[:product] && page.output_path.include?(page.metadata[:product]))
+                          # TODO: What do we do here if there's more than one product??
                           a['href'] = page.metadata[:browse] + '/' + page.output_path.split(page.metadata[:product]).last.gsub('index.html', '') + url
```

Comment:
No, this works, but we may want to look at it a bit more later.

On Tue, Oct 6, 2015 at 10:44 AM, Paul Robinson <notifications@github.com>
wrote:

> In _ext/jboss_developer.rb
> <https://github.com/redhat-developer/developers.redhat.com/pull/454#discussion_r41289755>
> :
>
> > @@ -117,17 +117,22 @@ def transform site, page, content
> >                      else
> >                        if (page.metadata[:browse].include?('blob') || page.metadata[:browse].include?('tree'))
> >                          if (page.metadata[:product] && page.output_path.include?(page.metadata[:product]))
> > +                          # TODO: What do we do here if there's more than one product??
> >                            a['href'] = page.metadata[:browse] + '/' + page.output_path.split(page.metadata[:product]).last.gsub('index.html', '') + url
>
> Is this a WiP? Do the todos need resolving before merge?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/redhat-developer/developers.redhat.com/pull/454/files#r41289755>
> .
>



-- 
Jason Porter
http://en.gravatar.com/lightguardjp